core/spec/models/spree/variant_spec.rb

```diff
@@ -45,6 +45,28 @@
       variant_with_same_sku = build(:variant, sku: variant.sku)
       expect(variant_with_same_sku).to be_invalid
     end
+
+    context "should have a price if master variant" do
```

Comment:
Just one nit re-reviewing: this sounds a bit off as a context title, isn't it?

Reply:
Fixed!